client-v2/src/components/FrontsEdit/CollectionOverview.tsx

```diff
@@ -61,6 +61,9 @@ const Container = styled.div`
 const TextContainerLeft = styled.div`
   flex: 1 1;
   font-size: 14px;
+  overflow-wrap: break-word;
```

Comment:
This fixes the overflowing exclamation marks in the overview for collections with long names